B827EBFFFE00F9D7.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE00F9D7",
+    "servers": [
+      {
+        "server_address": "us-west.thethings.network",
```

Comment:
I think that's not the correct URL for the us router, please check the list linked on the readme file of this repo.